WordPress/Classes/ViewRelated/Reader/DetailWebView/ReaderWebView.swift

```diff
@@ -19,25 +22,56 @@ class ReaderWebView: WKWebView {
         <!DOCTYPE html><html><head><meta charset='UTF-8' />
         <title>Reader Post</title>
         <meta name='viewport' content='initial-scale=1, maximum-scale=1.0, user-scalable=no'>
-        <link rel="stylesheet" type="text/css"
-              href="https://wordpress.com/calypso/fallback/async-load-blocks-reader-full-post.feb2901cfbd98579fa81.min.css">
+        <link rel="stylesheet" type="text/css" href="https://wordpress.com/calypso/reader-mobile.css">
```

Comment:
Thoughts on adding a some parameters to the URL?

Something like `platform=iOS`, `app-version=XX`, `os-version=XX`?

That way in the future if there's a bug with a specific version we can remotely fix it without having to do an update. 

A current example of this is the `<audio>` tag doesn't work on in the reader on iOS 11.4.

Reply:
We can do that. But I have a few concerns:

* This will change the CSS endpoint for every version we release. This is not a problem, but if a user updated the app and then later try to open an article while offline the CSS will be broken (because this file has never be loaded)
* We don't have the logic in the backend to deal with that parameters.
* I'd avoid adding conditionals and keeping the CSS as simple as possible. But who knows about the future?

Let me know what do you think. :)

Reply:
> This will change the CSS endpoint for every version we release. This is not a problem, but if a user updated the app and then later try to open an article while offline the CSS will be broken (because this file has never be loaded)

True. I think this may expose a deeper issue though. If someone upgrades to the new WP version but doesn't open it until they're offline, what happens for them? Is it worth considering adding a local fallback CSS file just in case?

We could bundle a version of the CSS in the app and have a CSS rule above the web version that points to it. When the web version is loaded it will override the local CSS properties. 

`<link rel="stylesheet" type="text/css" href="file:///Applications/WordPress.app/Resources/reader-mobile.css">` 

> We don't have the logic in the backend to deal with that parameters.

Currently it doesn't but the backend can be updated in the future if needed. 

> I'd avoid adding conditionals and keeping the CSS as simple as possible. But who knows about the future?

I agree, but speaking from experience we are at the whim of what WebKit supports in each OS version. There could be a bug that occurs only on iOS 12 when rendering a specific block.

Reply:
> True. I think this may expose a deeper issue though. If someone upgrades to the new WP version but doesn't open it until they're offline, what happens for them? Is it worth considering adding a local fallback CSS file just in case?

Offline support is yet to come, but if a user saved a post for later we will preload the whole post in a hidden webview (same strategy as Android uses).

Yes, if someone just upgraded, has saved articles, and tries to open a saved post while offline the CSS won't be available.

> Is it worth considering adding a local fallback CSS file just in case?
Maybe... but this has its own challenges:

* How to deal with the style conflicts? This is not a "fallback" but something that will always be loaded
* We would keep that updated over time?

> There could be a bug that occurs only on iOS 12 when rendering a specific block.

I agree. But how would that be different from fixing in the browser?

Reply:
I agree that this has its own technical challenges and is outside the scope of this PR. We can continue discussing this separately if needed.